src/openforms/formio/serializers.py

```diff
@@ -68,6 +68,12 @@ def apply_hidden_state(
             self._remove_validations_from_field(serializer_field)
 
     def _remove_validations_from_field(self, field: serializers.Field) -> None:
+        from .components.vanilla import EditGridField  # circular import
```

Comment:
I don't like this, but I'm not sure how I can leverage pattern matching here with kwargs that don't exist on `serializers.Field` :thinking: